timesheet_task/project_task.py

```diff
@@ -195,6 +195,26 @@ def _get_dummy_hr_analytic_timesheet_id(
         """Ensure all hr_analytic_timesheet_id is always False"""
         return dict.fromkeys(ids, False)
 
+    @api.multi
+    def on_change_account_id(self, account_id, context=None):
```

Comment:
OK, but here you are not calling super, so you are breaking inheritance chain then.